diff -urN nss-softokn-3.13.1.orig/mozilla/security/nss/cmd/manifest.mn nss-softokn-3.13.1/mozilla/security/nss/cmd/manifest.mn
--- nss-softokn-3.13.1.orig/mozilla/security/nss/cmd/manifest.mn        2012-01-26 23:50:28.000000000 +0300
+++ nss-softokn-3.13.1/mozilla/security/nss/cmd/manifest.mn     2012-01-26 23:51:02.000000000 +0300
@@ -42,11 +42,11 @@

 DIRS = lib  \
  bltest \
- fipstest \
  shlibsign \
  $(NULL)

 TEMPORARILY_DONT_BUILD = \
+ fipstest \
  $(NULL)

 # rsaperf  \
