Index: mozilla/security/nss/lib/freebl/stubs.c
===================================================================
RCS file: /cvsroot/mozilla/security/nss/lib/freebl/stubs.c,v
retrieving revision 1.11
diff -u -p -r1.11 stubs.c
--- mozilla/security/nss/lib/freebl/stubs.c	30 Mar 2011 01:20:12 -0000	1.11
+++ mozilla/security/nss/lib/freebl/stubs.c	20 Oct 2011 15:45:40 -0000
@@ -156,6 +156,7 @@ STUB_DECLARE(void,PR_DestroyCondVar,(PRC
 STUB_DECLARE(void,PR_Free,(void *ptr));
 STUB_DECLARE(char * ,PR_GetLibraryFilePathname,(const char *name,
 			PRFuncPtr addr));
+STUB_DECLARE(PRFileDesc *,PR_ImportFile,(PROsfd osfd));
 STUB_DECLARE(PRFileDesc *,PR_ImportPipe,(PROsfd osfd));
 STUB_DECLARE(void,PR_Lock,(PRLock *lock));
 STUB_DECLARE(PRCondVar *,PR_NewCondVar,(PRLock *lock));
@@ -307,6 +308,20 @@ PR_Open_stub(const char *name, PRIntn fl
 }
 
 extern PRFileDesc *
+PR_ImportFile_stub(PROsfd fd)
+{
+    int *lfd = NULL;
+
+    STUB_SAFE_CALL1(PR_ImportFile, fd);
+
+    lfd = PORT_New_stub(int);
+    if (lfd != NULL) {
+	*lfd = fd;
+    }
+    return (PRFileDesc *)lfd;
+}
+
+extern PRFileDesc *
 PR_ImportPipe_stub(PROsfd fd)
 {
     int *lfd = NULL;
@@ -566,6 +581,7 @@ freebl_InitNSPR(void *lib)
 {
     STUB_FETCH_FUNCTION(PR_Free);
     STUB_FETCH_FUNCTION(PR_Open);
+    STUB_FETCH_FUNCTION(PR_ImportFile);
     STUB_FETCH_FUNCTION(PR_ImportPipe);
     STUB_FETCH_FUNCTION(PR_Close);
     STUB_FETCH_FUNCTION(PR_Read);
diff -up ./mozilla/security/nss/lib/freebl/stubs.h.prelink ./mozilla/security/nss/lib/freebl/stubs.h
--- ./mozilla/security/nss/lib/freebl/stubs.h.prelink	2011-12-13 09:01:59.722227804 -0800
+++ ./mozilla/security/nss/lib/freebl/stubs.h	2011-12-13 09:07:40.134220235 -0800
@@ -77,6 +77,7 @@
 #define PR_DestroyLock  PR_DestroyLock_stub
 #define PR_Free  PR_Free_stub
 #define PR_GetLibraryFilePathname  PR_GetLibraryFilePathname_stub
+#define PR_ImportFile  PR_ImportFile_stub
 #define PR_ImportPipe  PR_ImportPipe_stub
 #define PR_Lock  PR_Lock_stub
 #define PR_NewCondVar PR_NewCondVar_stub
